Tribler/Core/Modules/MetadataStore/OrmBindings/channel_metadata.py

```diff
@@ -0,0 +1,304 @@
+import os
+from datetime import datetime
+from libtorrent import file_storage, add_files, create_torrent, set_piece_hashes, bencode, torrent_info
+
+from pony import orm
+from pony.orm import db_session
+
+from Tribler.Core.Modules.MetadataStore.serialization import ChannelMetadataPayload, CHANNEL_TORRENT
+from Tribler.Core.exceptions import DuplicateTorrentFileError, DuplicateChannelNameError
+
+CHANNEL_DIR_NAME_LENGTH = 60  # Its not 40 so it could be distinguished from infohash
+BLOB_EXTENSION = '.mdblob'
+CHUNK_SIZE_LIMIT = 1 * 1024 * 1024  # We use 1MB chunks as a workaround for Python's lack of string pointers
+
+
+def create_torrent_from_dir(directory, torrent_filename):
+    fs = file_storage()
+    add_files(fs, directory)
+    t = create_torrent(fs)
```

Comment:
This should use the piece size of `CHUNK_SIZE_LIMIT` and flags of `16 | 1 = 17` for mutable torrent support.

Reply:
As we discussed, we can optimize it later based on usage statistics.